MONOREPO_ROADMAP.md

```diff
@@ -0,0 +1,383 @@
+    note ~~~ sdk-client
+
+```
+
+## Packages
+
+This is not a comprehensive list of all packages in the monorepo - it does not include any datasources. These will all exist in `./packages/datasources`, and do not have any bearing on the structure of the [Evidence framework](#evidence-the-framework)
+
+### Library Packages
+
+#### `@evidence-dev/usql-duckdb`
+
+Provides the only open source driver for Universal SQL, using DuckDB WASM to deliver a fast query layer for NodeJS + Browser environments.
+
+This functionality was previously provided by `@evidence-dev/universal-sql`
+
+##### Does:
+- Provide standard functions for getting query columns and row counts
+
+##### Does Not:
+- Interact with DuckDB Native
+- Cache to or from the filesystem
+- Create `.parquet` files as part of the source evaluation process
+- Require being in a Vite project
+
+#### `@evidence-dev/telemetry`
+
+Allows Evidence to collect anonymous information about product usage
+
+##### Does:
+
+- Provide a set, documented list of informations that are collected
+- Provide a simple, clear way to opt out of telemetry
+
+##### Does Not:
+- Impact performance
+- Record any PII or sensitive information
+
+#### `@evidence-dev/vite`
+
+Responsible for providing a Vite plugin for converting a Vite app into an [Evidence App](#evidence-application)
+
+##### Does:
+- Source HMR, Query File HMR, etc
+- Caching, implicit API at `/_evidence`
+
+##### Does Not:
+- Copy files from a [Project](#evidence-project) to an [Application Shell](#application-shell)
+
+
+#### `@evidence-dev/sdk`
+
+Responsible for encapsulating all Evidence framework logic, contains the needed interfaces to create an [Evidence Appliction](#evidence-application).
+
+##### Does:
+- Implement stores using Svelte's store contract (without referencing Svelte directly)
+  - Consider using a tool like [nanostores](https://github.com/nanostores/nanostores)
+- Provides everything needed to turn a Vite project into an [Evidence Application](#evidence-application)
+  - This does not include framework-specific ergonomics such as those in [@evidence-dev/lang-svelte](#evidence-devlang-svelte)
+
+##### Does Not:
+- Depend on Svelte, React, or any specific framework
+
+##### Exports:
+
+###### `@evidence-dev/sdk/config`
+- Utilities for reading and writing `evidence.config.yaml` according to a passed-in Zod Schema
+
+###### `@evidence-dev/sdk/lib`
+- Library code used across environments
+- `EvidenceError`, `isDebug`, `fillMissingData`, `setTrackProxy`, etc.
+
+###### `@evidence-dev/sdk/lib/datasources`
+- Library code to ease the creation of datasource plugins
+- Previously `@evidence-dev/db-commons`
+
+###### `@evidence-dev/sdk/plugins/components`
+- Utilities for loading [Component Plugins](#component-plugin)
+
+###### `@evidence-dev/sdk/plugins/datasources`
+- Utilities for loading [Datasource Plugins](#datasource-plugin)
+- Utilities for evaluating [Datasources](#datasources)
+
+###### `@evidence-dev/sdk/plugins/app-shells`
```

Comment:
what is an application shell? is it the same as a layout?

shell is a confusing word to me as it can also mean terminal, so I might assume it have some relation to the CLI